cmd/docker/docker.go

```diff
@@ -247,6 +247,13 @@ func setHelpFunc(dockerCli command.Cli, cmd *cobra.Command) {
 			}
 		}
 
+		// FIXME(thaJeztah): need a better way for this; hiding the command here, so that it's present" by default for generating docs etc.
```

Comment:
Yes this looks good for now

Reply:
Let me squash the commits; it's not "perfect", but we can tweak more